app/components/common/committeeListing/index.js

```diff
@@ -36,41 +36,43 @@
 
       // ag-grid requires fixed widths in pixels. Calculating them here to take all available space
       var odcaGridElement = document.querySelectorAll('.odca-grid')[0];
-      var gridWidth = odcaGridElement.offsetWidth;
+      gridWidth = odcaGridElement.offsetWidth;
 
       // Convert resources to plain old objects to make ag-grid happy
       contributionsObjs = ctrl.contributions.map(function(resource) {
+        // Total listed contributions
         ctrl.committeeTotal += resource.Tran_Amt1;
+
         return {
-          Tran_NamL: resource.Tran_NamL,
+          Name: getFriendlyName(resource),
           Tran_Amt1: resource.Tran_Amt1,
           Tran_Date: resource.Tran_Date
         };
-      });      
+      });
 
       ctrl.gridOptions = {
         columnDefs: [
           {
             headerName: 'Contributor',
-            field: 'Tran_NamL',
+            field: 'Name',
             filter: 'text',
-            width: Math.round(gridWidth * .6 * 100)/100,
+            width: Math.round(gridWidth * 0.6 * 100) / 100,
             suppressMenu: true
           },
           {
             headerName: 'Amount',
             field: 'Tran_Amt1',
-            width: Math.round(gridWidth * .2 * 100)/100,
+            width: Math.round(gridWidth * 0.2 * 100) / 100,
             cellFormatter: function(params) {
-              return $filter('currency')(params.value, "$", 0);
+              return $filter('currency')(params.value, '$', 0);
```

Comment:
Also the dollar filter should really be in its own file or at least in a filter file. It's kind of buried right now.